eng/common/scripts/Invoke-GitHubAPI.ps1

```diff
@@ -1,3 +1,7 @@
+if ((Get-ChildItem -Path Function: | ? { $_.Name -eq "LogWarning" }).Count -eq 0) {
```

Comment:
What prompted this? Are there any normal cases were we would hit this?

Reply:
I gues I can just import it again. This is just for the case where someone is using the script directly and not through common.ps1.